dojo/finding_group/redis.py

```diff
@@ -0,0 +1,256 @@
+import base64
+import json
+import logging
+import os
+from dataclasses import asdict, dataclass, field
+from datetime import datetime
+from enum import StrEnum
+from functools import lru_cache
+from typing import Self
+
+import redis
+from django.conf import settings
+from django.utils.safestring import mark_safe
+
+from dojo.models import Finding
+
+logger = logging.getLogger(__name__)
+
+DD_TEST = os.getenv("DD_TEST", "False").lower() == "true"
+SEVERITY_ORDER = {
+    "Critical": 5,
+    "High": 4,
+    "Medium": 3,
+    "Low": 2,
+    "Info": 1,
+}
+USER_MODES_KEY = "finding_groups_user_modes"
+SYSTEM_CHANGE = "finding_groups_last_finding_change"
```

Comment:
Coudl this have a more clear name like LAST_FINDING_CHANGE or LAST_FINDING_UPDATE